.github/workflows/build.yml

```diff
@@ -35,6 +35,13 @@ jobs:
       - name: Update Build Date
         run: sed -i "s/@build-time@/`date +%Y%m%d%H%M%S`/" WeakAuras/Init.lua
 
+      - name: Flag Non-Experimental Build
+        run: |
+          sed -i -e "s/--@experimental@/--\[=====\[@experimental@/" WeakAuras/Init.lua \
```

Comment:
5 ='s, because bigwigs packager will use at most 4 ='s in its own @flag@ replacements